code/src/main/kotlin/com/expediagroup/sdk/core/configuration/ExpediaGroupDefaultClientConfiguration.kt

```diff
@@ -23,7 +23,7 @@ object ExpediaGroupDefaultClientConfiguration :
     override fun getEndpoint(): String = "https://api.expediagroup.com/"
     override fun getAuthEndpoint(): String = "${getEndpoint()}identity/oauth2/v3/token/"
     override fun getAuthenticationStrategy(): AuthenticationStrategy = AuthenticationStrategy.BEARER
-    override fun getRequestTimeout(): Long = Constant.INFINITE_TIMEOUT
+    override fun getRequestTimeout(): Long = -1
```

Comment:
Why? The constant is better for readability.

Reply:
Updated to:
```kotlin
override fun getRequestTimeout(): Long = Constant.ONE_HOUR_IN_MILLIS
```

Used the value of one hour because using `-1` can go wrong based on the internal HTTP client implementation (e.g. `Apache-4`, `OkHttp`, `Retrofit`, ...etc).